operating/view-dashboard.adoc

```diff
@@ -6,58 +7,100 @@ include::modules/common-attributes.adoc[]
+The *Deployments at most risk* widget provides a list of deployments that have been determined to have the highest risk, ##as determined by the risk priority.## The widget displays the deployment name, cluster and namespace, and risk priority. 
 
-You can select a deployment in this widget to get more details about the issues with that particular deployment.
-You can also select *View All* to view risks for all your deployments.
+You can perform the following actions in this widget:
 
-From here, you can select the separate deployments to see the details of the issues found with each particular deployment.
+* To view the *Risk* page for a deployment, click the deployment name. 
+* To view the *Network Graph* page for the deployment, click the cluster/namespace. 
+* To view all deployments at risk, select *View all*.
 
-[id="active-violations-by-time"]
-== Active Violations by Time
+[id="aging-images"]
+== Active aging images
 
-The *Active Violations by Time* widget shows the scale of number of violations by recent days, with the colors indicating the severity of those violations.
+The Active aging images widget shows the active images by age, allowing you to determine the number of images of each severity level that have not been updated for a selected period of time. 
 
-[id="configurable-dashboard-widgets"]
-== Configurable dashboard widgets
+The default view displays images that have low, medium, high, and critical severities by default age ranges. To configure different beginning and ending values for the image age bar graph items that are displayed, select *Options* and enter the desired values. Valid values are 1 through 365. Clear any box to collapse the age ranges and remove that item from the graph.
+
+For example, to view image counts by severity levels for 1-30, 30-60, and over 60 days, select *Options* and enter 1, 30, and 60 in the *Image age values* fields. Clear the last box. 
+
+[id="policy-violations-by-category"]
+== Policy violations by category
+
+The *Policy violations by category* widget provides a graphical representation of violations by pre-defined policy categories such as vulnerability management, security best practices, and privileges. This widget displays a color-coded bar graph of violations by severity level. You can hover over the bar for each category to view the number of violations. Click on a severity level at the bottom of the graph to add or remove that level from the display.  
+
+You can select the following options to further modify the display:
+
+* *Sort by*: 
+** *Severity*: Sorts by severity level. Policy categories with the largest number of criticals will be displayed first. 
+** *Total*: Sorts by the total number of violations of any severity.
+* *Policy Lifecycle*: ##You can configure the display to select policies that have violations during *Deployment*, *Runtime*, or *All* life cycles.##
```

Comment:
This is correct. We do not include an independent option for the "Build" lifecycle, but I honestly don't recall why not.